examples/rust/raw_mesh/src/main.rs

```diff
@@ -89,7 +88,7 @@ fn log_node(rec: &RecordingStream, node: GltfNode) -> anyhow::Result<()> {
     // Convert glTF objects into Rerun components.
     for (i, primitive) in node.primitives.into_iter().enumerate() {
         let mesh: Mesh3D = primitive.into();
-        rec.log(format!("{}/#{}", node.name, i), &mesh)?;
+        rec.log(format!("{}/{}", node.name, i), &mesh)?;
```

Comment:
Since we don't really handle path indices differently in the UI at the moment, this change doesn't really have any big effect, but there is an idea that the `#` prefix means "indices", e.g. of an array (like this). So for instance:

```rs
#[derive(rerun::Loggable)]
struct MyThing {
    array: Vec<…>,
    map: HashMap<int, …>,
}

rr.log("thing", MyThing {
    array: [a, b],
    map: {42: foo, 1337: bar},
});
```

would be logged the paths:

* `thing/array/#0`
* `thing/array/#1`
* `thing/map/42`
* `thing/map/1337`

anyway, this is all less than half-baked so 🤷

Reply:
I removed the `#`s because that resulted in WARN spam with the new entity path parsing rules.
![image](https://github.com/rerun-io/rerun/assets/2910679/86b5befd-1f29-4cec-9301-46e6d07e4dd3)